.buildbot.sh

```diff
@@ -94,6 +94,33 @@ for i in $(seq 10); do
     YKD_NEW_CODEGEN=1 RUST_TEST_SHUFFLE=1 cargo test
 done
 
+# Test with LLVM sanitisers
+RUSTFLAGS="-Z sanitizer=address" cargo test --target x86_64-unknown-linux-gnu
+# The thread sanitiser has a number of false positives by default, so we need a
+# suppression file to avoid those overwhelming us. This does mean that we might
+# suppress some true positives, but there's little we can do about that.
+suppressions_path=`mktemp`
+cat << EOF > $suppressions_path
```

Comment:
If we decide to use the sanitisers, I'd just put the suppression stuff into a file and import it rather than generating it each time.

Reply:
The advantage of doing it like this is that test + suppression have to evolve together. [Whether that turns out to be a good idea in the medium term isn't clear to me yet.]

Reply:
You mean the locality is better?

Reply:
Yes, while we're in the experimental phase.